src/include/miopen/conv/invokers/flexgemm.hpp

```diff
@@ -0,0 +1,40 @@
+/*******************************************************************************
+*
+* MIT License
+*
+* Copyright (c) 2020 Advanced Micro Devices, Inc.
+*
+* Permission is hereby granted, free of charge, to any person obtaining a copy
+* of this software and associated documentation files (the "Software"), to deal
+* in the Software without restriction, including without limitation the rights
+* to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+* copies of the Software, and to permit persons to whom the Software is
+* furnished to do so, subject to the following conditions:
+*
+* The above copyright notice and this permission notice shall be included in all
+* copies or substantial portions of the Software.
+*
+* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+* IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+* FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+* AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+* LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+* OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+* SOFTWARE.
+*
+*******************************************************************************/
+#ifndef GUARD_MIOPEN_FLEXGEMM_HPP
+#define GUARD_MIOPEN_FLEXGEMM_HPP
+
+#include <miopen/invoker.hpp>
+#include "../../../../solver/flexgemm/include/flexgemm_param.hpp"
```

Comment:
Move flexgemm_param.hpp to common directory which is holding header files or add the solver/flexgemm/include to compiler including directory.

Reply:
> Move flexgemm_param.hpp to common directory which is holding header files or add the solver/flexgemm/include to compiler including directory.

It in compiler including directory of src/CMakeLists.txt:
`list(APPEND MIOpen_Source
    solver/flexgemm/include/flexgemm_param.hpp
    solver/flexgemm/source/flexgemm_param.cpp
    solver/conv_flexgemm.cpp
    include/miopen/conv/invokers/flexgemm.hpp
    conv/invokers/flexgemm.cpp
)`

Reply:
You don't have to add header files to source list. The only benefit I know if is that it would be visible in the solution explorer of visual studio. C++ doesn't require anything to include file beside writing #include with "", because it allows relative paths.